notebooks/community/managed_notebooks/chicago_taxi_fare_prediction/chicago_taxi_fare_prediction_notebook.ipynb

```diff
@@ -0,0 +1,1440 @@
+{
```

Comment:
<p>should follow notebook template for setting PROJECT</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>resolved</p>